### PR TITLE
Fix 907: Add Enrollments to the Genesis block

### PR DESCRIPTION
**Pull request by Geod24**
For real this time. See each commit.

### CODE REVIEW

File: source/agora/consensus/Genesis.d [hunk 1]
```diff
@@ -94,6 +95,68 @@ private immutable Block UnitTestGenesisBlock = {
...
+            Enrollment(
+                Hash(`0xdb3931bd87d2cea097533d82be0a5e36c54fec8e5570790c3369bd8300c65a0` ~
+                     `3d76d12a74aa38ec3e6866fd64ae56091ed3cbc3ca278ae0c8265ab699ffe2d85`),
+                Hash(`0x23dc305988b8ff32232256192f2350fe8cde4f54c60ca266c444489b9081702` ~
+                     `0d3c38148f877b8b16a80e46ce35e1a3e66a5309786282b816627961e1dcb088e`),
+                1008,
+                Signature(`0x0e251a27c71664bd6105b5f3817c5833971f13a57c02457fcc1c7f7cc937553` ~
+                          `55c71e74382a24b7e644d32b0306fe3cf14ecd7de5635c70aa592f4721aa74fe2`)
+            ),
+            //Enrollment(utxo_key, random_seed, 1008, sig),
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Genesis.d]: leftover?

File: source/agora/consensus/Genesis.d [hunk 2]
```diff
@@ -94,6 +95,68 @@ private immutable Block UnitTestGenesisBlock = {
         prev_block:  Hash.init,
         height:      Height(0),
         merkle_root: UnitTestGenesisMerkleTree[$ - 1],
+        enrollments: [
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Genesis.d:98]: So there are 6 enrollments in this genesis block?

Comment (Geod24) [reply source/agora/consensus/Genesis.d:98]: Yep

File: source/agora/consensus/Genesis.d (hunk 1 above)
Comment (Geod24) [reply source/agora/consensus/Genesis.d]: Yep

Comment (Geod24) [reply source/agora/consensus/Genesis.d]: Fixed

**Comment by AndrejMitrovic** [comment]
The + in the PR really shows that this PR is mixing two different functionalities. It might be better off in its own PR.

**Comment by AndrejMitrovic** [comment]
But I'm guessing this PR needs it, since you designed it this way. Oh well..

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by Geod24** [comment]
> The + in the PR really shows that this PR is mixing two different functionalities. It might be better off in its own PR.

Here you go: #989

**Comment by Geod24** [comment]
Rebased on v0.x.x

**Pull request merged**